target/linux/realtek/dts/rtl931x.dtsi

```diff
@@ -205,6 +214,10 @@
 		pinmux_disable_sys_led: disable_sys_led {
 			pinctrl-single,bits = <0x0 0x0 0x100>;
 		};
+
+		pinmux_gpio_mdio_en: gpio-mdio-en {
```

Comment:
why is there now this change which adds `mdio-aux` and `gpio-mdio-en` for all rtl931x devices in a commit which is supposed to add only PSX28/ESX28 support?

And it is also using a compatibility string which is for a completely different SoC. Please clean this up and make it a separate commit for adding `realtek_aux_mdio_driver` to the driver and the shared `rtl931x.dtsi`. See d45890f6cd508256356f99c11b8a744a30d99cf4 for a commit which adds it tot he rtl930x.dtsi